### DISCUSSION TITLE
공학글 - 좋은 글, 나쁜 글, Survey

### DISCUSSION THREAD

**Discussion by 1jeongg** [BDD-CLUB/challenge-every-week-posting#198] [4. 기타 글 모음]
새롭게 Github 블로그를 만들어봤습니다!

- 좋은글: [프로그래밍의 종말](https://1jeongg.github.io/good-writing/)
- 나쁜글: [나쁜 글 == 읽기 힘든 글](https://1jeongg.github.io/bad-writing/)
- Survey: [결혼 준비 웹사이트 비교](https://1jeongg.github.io/wedding-website-survey/)

**Comment by pnudpwls1103** [comment]
블로그 너무 예쁘네요!!
